cocos/game/director.ts

```diff
@@ -691,10 +701,11 @@ export class Director extends EventTarget {
                 for (let i = 0; i < this._systems.length; ++i) {
                     this._systems[i].postUpdate(dt);
                 }
+
+                this.emit(Director.EVENT_UPDATE_UI);
```

Comment:
UIEvent timing should not be place in `not-pause` branch. UI should ignore director‘s pause.